### ISSUE TITLE
max_conns and max_idle_conns are not removed from DSN

### ISSUE THREAD

**Issue by vesvalo** [ory/hydra#1327]
The settings max_conns and max_idle_conns are not removed from DSN string when hydra connects to Postgres.

If DATABASE_URL is set like this:
```
DATABASE_URL="postgres://pghost:5432/hydra?sslmode=disable&max_conns=20&max_idle_conns=4"
```
Hydra can't connect to the Postgres.
Postges logs:
```
2019-03-27 13:22:22.615 GMT [41732] FATAL:  unrecognized configuration parameter "max_idle_conns"
2019-03-27 13:22:37.629 GMT [41734] FATAL:  unrecognized configuration parameter "max_conns"
```

**Comment by aeneasr** [comment]
Please use the issue templates. Crucial information is missing, like software version, command you executed, and so on. This issue will auto-close otherwise.

**Comment by vesvalo** [comment]
Command to run:   
```
export DATABASE_URL="postgres://pghost:5432/hydra?sslmode=disable&max_conns=20&max_idle_conns=4"
hydra serve all --dangerous-force-http
````
Environment, software version: kubernetes, docker image hydra:v1.0.0-rc.6_oryOS.10-alpine

The issue was addressed in the #464, but that fix is not present in the repository now.
